pkgs/development/tools/misc/texinfo/common.nix

```diff
@@ -58,7 +62,7 @@ stdenv.mkDerivation {
     && !stdenv.isDarwin
     && !stdenv.isSunOS; # flaky
 
-  checkFlags = lib.optionals (!stdenv.hostPlatform.isMusl) [
+  checkFlags = lib.optionals (!stdenv.hostPlatform.isMusl && lib.versionOlder version "7.0")) [
```

Comment:
```suggestion
  checkFlags = lib.optionals (!stdenv.hostPlatform.isMusl && lib.versionOlder version "7") [
```

Reply:
Thanks for the suggestion, I saw the faulty duplicate parenthesis and addressed this already. Will remove the `.0` from the version as you suggested and rebase and force push.